MIGRATION.md

```diff
@@ -10,9 +10,9 @@
 
 We updated the React DatePicker from 1.X to 3.X. The library no longer uses moment. Instead it uses DateFns as a dependency for date management. Therefore, the Date Input no longer uses moment object for the value, but a Javascript [Date object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date). Also, the onChange callback return a Date value instead of a moment Value.
 
-The `locale` param also changed. It is a string which set the language of the component. You can find the full list on [DateFns's website](https://date-fns.org/v2.0.0-alpha.18/docs/I18n#supported-languages). This feature uses the [dynamic import](https://reactjs.org/docs/code-splitting.html#import) in order to import only the locales needed by the user. Be sure that your project is able to manage that with an approriate tool (like Webpack).
+The `locale` param also changed. It is a string which sets the language of the component. You can find the full list on [DateFns's website](https://date-fns.org/v2.0.0-alpha.18/docs/I18n#supported-languages). This feature uses the [dynamic import](https://reactjs.org/docs/code-splitting.html#import) in order to import only the locales needed by the user. Make sure that your project is compatible that with an approriate tool (_create-react-app_ does it out-of-the-box).
```

Comment:
« Make sure that your project is compatible ~~that~~ with an appropriate tool »

Reply:
Changed the sentence to 
> Make sure that your project is compatible with code splitting. (_create-react-app_ does it out-of-the-box).

Does it sound better? 🤔

Reply:
Pour moi c'est plus clair, la version actuelle.
Code splitting me donne une impression de complexité.

Reply:
Yes, without the point after "splitting" :grin: